grails-app/migrations/views/order-summary-view.sql

```diff
@@ -13,7 +13,7 @@ CREATE OR REPLACE VIEW order_item_status AS
             product.product_code        AS product_code,
             order_item.quantity * order_item.quantity_per_uom    AS quantity_ordered, -- to compare with shipped quantity which is already multiplied by qty per uom
             CASE
-                WHEN shipment.current_status IN ('SHIPPED', 'PARTIALLY_RECEIVED', 'RECEIVED') THEN SUM(shipment_item.quantity)
+                WHEN COALESCE(shipment.current_status) IN ('SHIPPED', 'PARTIALLY_RECEIVED', 'RECEIVED') THEN SUM(shipment_item.quantity)
```

Comment:
What does COALESCE do here?

Reply:
You are right this is wrong approach. I added the shipment into the group by (but it will need testing on the multiple shipments per order item case).